master/networking/sidecar-acceleration.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Accelerating Istio network performance
+---
+
+### Big picture
+
+Use Calico to accelerate network performance of routing network traffic via Istio Envoy sidecar.
+
+> **Warning!** This feature is experimental and should not be used in production clusters. It uses a recent Linux kernel feature (eBPF SOCKMAP), which our testing confirms requires upstream kernel enhancements to reliably and securely support production clusters. We are contributing fixes to the kernel where needed.
+{: .alert .alert-danger }
+
+### Value
+
+Istio directs all application network traffic through an Envoy sidecar in each pod, which introduces network overhead for all traffic. Calico can greatly reduce this network overhead by automatically optimizing the Linux network path for this traffic.
+
+### Features
+
+This how-to guide uses the following Calico features:
+
+**Felix configuration** with **SidecarAccelerationEnabled** configuration option. 
+
+
+### Concepts
+
+#### Sidecar acceleration
+
+The Sidecar acceleration process bypasses several layers of kernel networking, allowing data to flow between the sockets unobstructed and to accelerate. This makes the Envoy proxy (sidecar) to container network path as fast and efficient as possible. 
```

Comment:
"unobstructed and to accelerate" seems wrong - accelerator what?

Maybe just drop the "and to accelerate" bit?

Reply:
removed.